src/profilers/profile-serializer.ts

```diff
@@ -117,30 +111,36 @@ export function serializeTimeProfile(
     if (f !== undefined) {
       return f;
```

Comment:
It's a bit strange that the coverage says this line is not covered. We don't have any two samples that point to a same function?

Reply:
I think this is because locationMap and functionMap use the same key (callUid). So, this code ends up basically being unreachable, because if there is a function, then there is a location, so the location will be returned and getFunction() won't be called.
This isn't problematic now, since there isn't any finer profile resolution than function-level resolution. However, the one-to-one correspondence between functions and locations isn't great, so it's something that I should address in a future PR -- I'll file an issue for it.